index.html

```diff
@@ -118,9 +119,70 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
                   <li class="language">javascript</li>
                   </ul>
               </div>
-              <div><button class="see-project" type="button">See project</button></div>
+              <div ><button id="see-project1" class="see-project1"  type="button">See project</button></div>
           </div>
+        </div>
+
+ <!--###################################### start window popup 1 ~~################################### -->
+ <div class="background-popup">
+ <div class="popup1">
+   
+    <div class="Works1popup1">
+        <div class="toggle1">
+            <i class="fas fa-times fermer1"></i>         
+        </div>
+     <div><h2 class="textpop" ></h2></div>
+     
+     <div class="flex1popup1">
+      <div class="factorypopup1"></div>
+      <div class="roundpoppup1"></div>
+      <div class="flex1-langpopup1"></div>
+      <div class="roundpoppup1"></div>
+      <div class="flex1-langpopup2"></div>
+  </div>
+
+    
+       <div class="img-1popup1"><img class="fontimg1popup1"  width="100%"  height="100%" src="" alt=""><img class="fontimg2popup1"  width="100%"  height="100%" src="" alt=""></div>  
```

Comment:
[OPTIONAL] 
- These images look stretched. I would suggest making the look unstretched like in the original design
### Your Design 
![image](https://user-images.githubusercontent.com/45460192/225720450-ceb7e3ac-45cc-4496-8116-6ae9c1259658.png)
### Original
![image](https://user-images.githubusercontent.com/45460192/225720552-24698db2-08b3-4589-b5df-67cff7f7376a.png)